packages/mllp-server/src/config.ts

```diff
@@ -0,0 +1,11 @@
+import { Config as CoreConfig } from "@metriport/core/util/config";
+import { getEnvVar } from "@metriport/shared/common/env-var";
+
+export class Config extends CoreConfig {
+  static getSentryDSN(): string | undefined {
+    return getEnvVar("SENTRY_DSN");
+  }
+  static getVersion(): string | undefined {
+    return getEnvVar("METRIPORT_HL7_SERVER_VERSION");
```

Comment:
How's this being set on the cloud env? I assume a git hash like we do on the OSS API and Dash?

Reply:
Yep a git hash. Tried to clarify the naming. Hash gets passed down through props as a version field and makes its way into the container as RELEASE_SHA.

[top level source of value](https://github.com/metriport/metriport/blob/df27c72748f3b26660e3940f2b7d165e3b7ed573/packages/infra/bin/infrastructure.ts#L26) in infra that trickles down to the [newly named env var](https://github.com/metriport/metriport/blob/e275fde86285f191b21a3ed8bb9e4be5fff10332/packages/mllp-server/src/config.ts#L9)